### PR TITLE
feat(sessions): Change bottom two cards to Stability Score and Apdex

### LINKED ISSUE #96: feat(sessions): Change bottom two cards to Stability Score and Apdex

### PR DESCRIPTION
**Pull request by denrase**
Closes #96 
Relates to #81

### CODE REVIEW

File: lib/utils/stability_score.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      } else if (group?.by?.sessionStatus == SessionStatus.crashed) {
+        crashed = group?.totals?.sumSession;
+      }
+    }
+    if (healthy == null || errored == null || abnormal == null || crashed == null) {
+      return null;
+    }
+    final sum = healthy + abnormal + crashed + errored;
+    if (sum == 0) {
+      return 100;
```
Comment (matejminar) [inline lib/utils/stability_score.dart:26]: If the sum is 0 should we assume that there are no sessions in the time period and therefore stability is not 100% but rather `-`?

File: lib/screens/health/health_screen.dart [hunk 2]
```diff
@@ -137,68 +136,18 @@ class _HealthScreenState extends State<HealthScreen> {
                           Row(
                             children: [
                               HealthCard(
-                                  title: 'Crash Free Users',
-                                  value: viewModel.projects[_index].release?.crashFreeUsers,
-                                  change: 0.04
-                              ), // TODO(denis): Use delta from api, https://github.com/getsentry/sentry-mobile/issues/11
+                                  title: 'Stability Score',
```
Comment (matejminar) [inline lib/screens/health/health_screen.dart:139]: We are renaming this to `Crash Free` on web for now - I think we should do the same here.

**Review by matejminar — COMMENTED** [review]
How do we round stability score? We have this handy util on web where it displays max 3 decimal places if the score is higher than 95%, otherwise rounded to the whole numbers.

https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/releases/utils/index.tsx

**Comment by denrase** [comment]
@matejminar Not seeing too much data for the apdex value with the short 12h window. Please give me feedback once the next TestFlight version is out if the value is useful, as i'm sure you have projects with more traffic.

**Comment by denrase** [comment]
@matejminar We always show two decimal points for now, but i will change it to the web approach.

**Pull request merged**